crypto/keyring/record.go

```diff
@@ -59,6 +60,16 @@ func NewMultiRecord(name string, pk cryptotypes.PubKey) (*Record, error) {
 	return newRecord(name, pk, recordMultiItem)
 }
 
+// GetMultisigPubKey fetches a public key of the multi type record
+func (k *Record) GetMultisigPubKey() (*multisig.LegacyAminoPubKey, error) {
```

Comment:
I don't think this is a good API on Record: there should only be a method to get the PubKey.

Then, in tx_sign.go, we can cast `.GetPubKey().(*multisig.LegacyAminoPubKey)`